### PR TITLE
Add New Targets to build

### PR DESCRIPTION
**Pull request by jdauphant**
## Type of change

- [ ] Feature
- [ ] Bugfix
- [x] Technical
- [ ] Other :

## Content

Add new target to automatically build

## Motivation and context

I need dev APKs for testing purpose.
I want to test VoIP to

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [ GplayBtchapWithoutvoipWithpinning, GplayTchapWithoutvoipWithpinning, FdroidBtchapWithoutvoipWithoutpinning, FdroidTchapWithoutvoipWithoutpinning ]
+        target: [ GplayBtchapWithoutvoipWithpinning, GplayTchapWithoutvoipWithpinning, FdroidBtchapWithoutvoipWithoutpinning, FdroidTchapWithoutvoipWithoutpinning, GplayDevTchapWithoutvoipWithpinning, FdroidDevTchapWithoutvoipWithoutpinning, GplayBtchapWithvoipWithpinning]
```
Comment (jdauphant) [inline .github/workflows/build.yml:55]: :thinking: I may need to pull that in debug in fact

Comment (jdauphant) [reply .github/workflows/build.yml:55]: Let's do that

Comment (Florian14) [inline .github/workflows/build.yml:55]:
```suggestion
        target: [ GplayBtchapWithoutvoipWithpinning, GplayTchapWithoutvoipWithpinning, FdroidBtchapWithoutvoipWithoutpinning, FdroidTchapWithoutvoipWithoutpinning, GplayDevTchapWithoutvoipWithpinning, FdroidDevTchapWithoutvoipWithoutpinning, GplayBtchapWithvoipWithpinning ]
```

**Review by Florian14 — APPROVED** [review]
LGTM but that will increase the CI duration and the risk of failure while generating the apks (there are 4 APKs for each variant)

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -52,7 +52,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [ GplayBtchapWithoutvoipWithpinning, GplayTchapWithoutvoipWithpinning, FdroidBtchapWithoutvoipWithoutpinning, FdroidTchapWithoutvoipWithoutpinning ]
+        target: [ GplayBtchapWithoutvoipWithpinning, GplayTchapWithoutvoipWithpinning, FdroidBtchapWithoutvoipWithoutpinning, FdroidTchapWithoutvoipWithoutpinning, GplayDevTchapWithoutvoipWithpinning, FdroidDevTchapWithoutvoipWithoutpinning, GplayBtchapWithvoipWithpinning ]
```
Comment (yostyle) [inline .github/workflows/build.yml:55]: DevTchap should not be used on main branch
```suggestion
        target: [ GplayBtchapWithoutvoipWithpinning, GplayTchapWithoutvoipWithpinning, FdroidBtchapWithoutvoipWithoutpinning, FdroidTchapWithoutvoipWithoutpinning, GplayBtchapWithvoipWithpinning ]
```

**Review by yostyle — CHANGES_REQUESTED** [review]
DevTchap should be used to debug instead of Btchap except to test VoIP and pinning.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [ GplayBtchapWithoutvoipWithpinning, FdroidBtchapWithoutvoipWithoutpinning ]
+        target: [ GplayBtchapWithoutvoipWithpinning, FdroidBtchapWithoutvoipWithoutpinning, GplayDevTchapWithoutvoipWithpinning, FdroidDevTchapWithoutvoipWithoutpinning, GplayBtchapWithvoipWithpinning ]
```
[the remark this answers is not in the document]
Comment (jdauphant) [reply .github/workflows/build.yml]: Good for me

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [ GplayBtchapWithoutvoipWithpinning, FdroidBtchapWithoutvoipWithoutpinning ]
+        target: [ GplayDevTchapWithoutvoipWithoutpinning, FdroidDevTchapWithoutvoipWithoutpinning, GplayBtchapWithvoipWithpinning ]
```
Comment (jdauphant) [inline .github/workflows/build.yml:23]: @yostyle :
```suggestion
        target: [ GplayDevTchapWithoutvoipWithoutpinning, FdroidDevTchapWithoutvoipWithoutpinning ]
```

**Review by yostyle — APPROVED** [review]

**Pull request merged**